docs/ref/request-response.txt

```diff
@@ -1314,6 +1327,11 @@ Attributes
     This is useful for middleware needing to wrap
     :attr:`StreamingHttpResponse.streaming_content`.
 
+.. attribute:: StreamingHttpResponse.getvalue()
+
+    Returns the value of :attr:`StreamingHttpResponse.streaming_content`. This method makes
+    an :class:`StreamingHttpResponse` instance a stream-like object.
```

Comment:
I'm not sure if we should document this :thinking: 
If we do, maybe...
```suggestion
.. method:: StreamingHttpResponse.getvalue()

    The iterated response content, concatenated into a bytestring encoded
    according to :attr:`HttpResponse.charset`.
```

Reply:
Thank you for letting me know. After giving it some more thought, it doesn't seem appropriate for the current situation. I'll remove that part